src/main/resources/es-AR.yml

```diff
@@ -8,7 +8,30 @@ es-AR:
       street_suffix: [Avenida, Avda., Camino, Pasaje, Paseo]
       secondary_address: ['Esc. #', 'Puerta blanca', 'Porton verde', 'Reja negra', 'Pasillo al fondo', 'frente', 'piso #, depto. A']
       # Though these are US-specific, they are here (in the default locale) for backwards compatibility
```

Comment:
Not related to this PR perhaps, but is this comment still relevant? If so, I don't get it :)